### PR TITLE
add user view /people/add GET

### PR DESCRIPTION
**Pull request by JMurphyWeb**
Relies on https://github.com/postgres-plugin/people/pull/46#discussion_r91946375

issue #442

Add user view

endpoint: people/add GET

still need to
- [x] add tests
- [x] add to readme

### CODE REVIEW

**Comment by JMurphyWeb** [comment]
@mk4111 I think this can be reviewed now along with the PR linked in the description

File: server/handlers/people/add-view.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+var Hoek = require('hoek');
+var helpers = require('../helpers.js');
+
+function removeLinked (org) {
+  return org.active_primary_user === null;
```
Comment (mk4111) [inline server/handlers/people/add-view.js]: can we make this camel case?

File: server/handlers/people/add-view.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+module.exports = function (request, reply, source, joiErr) {
+  var error = helpers.errorOptions(joiErr);
+  var loggedIn = request.auth.credentials;
+  var permissions = helpers.getPermissions(loggedIn, 'scope', 'admin');
+  var unlinkedActiveOrgs, options;
+
+  request.server.methods.pg.organisations.getActive(function (pgErr, orgs) {
+    Hoek.assert(!pgErr, 'database error');
+
+    unlinkedActiveOrgs = orgs.filter(removeLinked);
```
Comment (mk4111) [inline server/handlers/people/add-view.js]: Do we need to invoke `removeLinked` with orgs? Or does this know to use orgs as a global var?

File: server/handlers/people/add-view.js (hunk 1 above)
Comment (SimonLab) [inline server/handlers/people/add-view.js]: For camelCase we will need to change the challenges plugin. This is also the value returns by pg which is not in camelCase.

File: server/handlers/people/add-view.js (hunk 2 above)
Comment (SimonLab) [inline server/handlers/people/add-view.js]: I'm not sure to understand fully your question.
We can either write an anonymous fuction in filter:
```
orgs.filter(function (org) {
 return org.active_primary_user === null;
})
```
Or extract the function like it is currently with removeLinked

File: server/handlers/people/add-view.js (hunk 1 above)
Comment (mk4111) [inline server/handlers/people/add-view.js]: Ah, so this gets directly put into the plugin. I thought it was specifically for within the app. I understand

**Comment by SimonLab** [comment]
Thanks @JMurphyWeb and @mk4111 for the review, :tada:

**Pull request merged**